packages/wallet-comm/src/wallet-functions.ts

```diff
@@ -0,0 +1,163 @@
+import { Channel } from 'fmg-core';
+import { INITIALIZATION_SUCCESS, INITIALIZATION_FAILURE, CHANNEL_OPENED, ChannelOpened, FUNDING_FAILURE, FUNDING_SUCCESS, FundingResponse, SIGNATURE_FAILURE, SIGNATURE_SUCCESS, SignatureResponse, VALIDATION_SUCCESS, VALIDATION_FAILURE, ValidationResponse, messageRequest, MESSAGE_REQUEST, SHOW_WALLET, HIDE_WALLET, CONCLUDE_FAILURE, CONCLUDE_SUCCESS } from './interface/from-wallet';
+import { INITIALIZE_REQUEST, openChannelRequest, initializeRequest, fundingRequest, signatureRequest, validationRequest, receiveMessage, concludeChannelRequest } from './interface/to-wallet';
+import BN from 'bn.js';
+import { WalletEventListener } from '.';
+
+
+export function createWalletIFrame(iframeId: string, walletUrl: string): HTMLIFrameElement {
+  const iFrame = document.createElement("iframe");
+  iFrame.src = walletUrl;
+  iFrame.id = iframeId;
+  iFrame.style.display = 'none';
+  iFrame.style.position = 'absolute';
+  iFrame.style.left = '0px';
+  iFrame.style.right = '0px';
+  iFrame.style.bottom = '0px';
+  iFrame.style.top = '0px';
+  iFrame.width = '0';
+  iFrame.height = '0';
+  iFrame.style.zIndex = '9999';
+  iFrame
```

Comment:
Is ts linting enabled for this code?

Reply:
Not yet, that's still a TODO (This one particular case has been cleaned up)